frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -397,10 +397,14 @@ class Worksheet extends React.Component {
             this.setState({ openedDialog: null, deleteItemCallback: null });
             return;
         }
-        if (cmd_type === 'deleteItem') {
```

Comment:
What was deleteItem used for earlier? Do we not need a handler for that case anymore?

Reply:
previously `deleteitem` refers to `deleteMarkdown` only. I think the naming is ambiguous since we now have `delete Schema` as well, so I rename it to be `deleteMarkdown` & `deleteSchema`